### PR TITLE
Show address line 2 on Vet Center pages

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/26544

### CODE REVIEW

File: src/site/layouts/tests/vet_center/vet_center.drupal.unit.spec.js [hunk 1]
```diff
@@ -41,7 +52,7 @@ describe('Vet Center Main Page', () => {
     ).to.equal(1);
   });
 
-  it('renders prepare for your visit field-cc-vet-call-center', () => {
+  it('renders prepar(e for your visit field-cc-vet-call-center', () => {
```
Comment (gescobardev) [inline src/site/layouts/tests/vet_center/vet_center.drupal.unit.spec.js:55]: You accidentally added a (

File: src/site/includes/vet_centers/address_phone_image.liquid [hunk 2]
```diff
@@ -26,6 +26,9 @@
             <address class="vads-u-margin-bottom--0">
                 <div>{{ vetCenter.fieldAddress.organization }}</div>
                 <div>{{ vetCenter.fieldAddress.addressLine1 }}</div>
+                {% if vetCenter.fieldAddress.addressLine1  %}
```
Comment (gescobardev) [inline src/site/includes/vet_centers/address_phone_image.liquid:29]: I believe you meant to check `{% if vetCenter.fieldAddress.addressLine2  %}`

**Review by gescobardev — CHANGES_REQUESTED** [review]
Looks solid! Just have a couple comments

File: src/site/includes/vet_centers/address_phone_image.liquid (hunk 2 above)
Comment (mpelzsherman) [reply src/site/includes/vet_centers/address_phone_image.liquid:29]: Great catch, thanks!

**Review by brangi — APPROVED** [review]

**Review by gescobardev — APPROVED** [review]

**Pull request merged**